packages/mdc-slider/foundation.js

```diff
@@ -273,18 +303,28 @@ export default class MDCSliderFoundation extends MDCFoundation {
 
       this.adapter_.registerBodyInteractionHandler(moveEvt, moveHandler);
       UP_EVENTS.forEach((type) => this.adapter_.registerBodyInteractionHandler(type, upHandler));
-      this.setValueFromEvt_(evt, getPageX);
+      this.setValueFromEvt_(evt, /** @type {function(!Event): !number} */ (getPageX));
     };
 
     return downHandler;
   }
 
+  /**
+   * Sets the slider value from an event
+   * @param {!Event} evt
+   * @param {!function(!Event): !number} getPageX
```

Comment:
`function` types are non-null by default, so you can remove the `!` qualifier before `function`